students/AndrewKim/lesson2hw.py

```diff
@@ -0,0 +1,85 @@
+
+
+#part 2
+def print_grid(n):
+	#first (+-)row
+	print('+', end=' ')
```

Comment:
make sure your code editor is set to 4 spaces exactly per pep8 style guidelines.